src/modules/CSADigitizer/CSADigitizerModule.cpp

```diff
@@ -80,6 +82,10 @@ CSADigitizerModule::CSADigitizerModule(Configuration& config, Messenger* messeng
         clockToT_ = config_.get<double>("clock_bin_tot");
     }
 
+    // Synchronize the clock binning to global simulation time
+    sync_sim_time_ = config_.get<bool>("sync_sim_time");
```

Comment:
Now you're setting a different default than what you are reading

Reply:
That was a stupid oversight, hopefully fixed everywhere now.